group_vars/all.yml.sample

```diff
@@ -76,68 +79,66 @@ dummy:
 #  - hdparm
 #  - python-setuptools
 
+# Whether or not to install the ceph-test package.
+#ceph_test: false
+
 # Enable the ntp service by default to avoid clock skew on
 # ceph nodes
 #ntp_service_enabled: true
 
-# Whether or not to install the ceph-test package.
-#ceph_test: False
-
-## Configure package origin
-#
-#ceph_origin: 'upstream' # or 'distro' or 'local'
-# 'distro' means that no separate repo file will be added
-# you will get whatever version of Ceph is included in your Linux distro.
-# 'local' means that the ceph binaries will be copied over from the local machine
+# Set uid/gid to default '64045' for bootstrap directories.
+# '64045' is used for debian based distros. It must be set to 167 in case of rhel based distros.
+# These values have to be set according to the base OS used by the container image, NOT the host.
+#bootstrap_dirs_owner: "64045"
+#bootstrap_dirs_group: "64045"
 
-# LOCAL CEPH INSTALLATION (ceph_origin==local)
-#
-# Path to DESTDIR of the ceph install
-#ceph_installation_dir: "/path/to/ceph_installation/"
-# Whether or not to use installer script rundep_installer.sh
-# This script takes in rundep and installs the packages line by line onto the machine
-# If this is set to false then it is assumed that the machine ceph is being copied onto will already have
-# all runtime dependencies installed
-#use_installer: false
-# Root directory for ceph-ansible
-#ansible_dir: "/path/to/ceph-ansible"
+# This variable determines if ceph packages can be updated.  If False, the
+# package resources will use "state=present".  If True, they will use
+# "state=latest".
+#upgrade_ceph_packages: False
 
 #ceph_use_distro_backports: false # DEBIAN ONLY
 
-# STABLE
-########
+
+###########
+# INSTALL #
+###########
+
+#ceph_repository: dummy
+
+# ORIGIN SOURCE
+#
+# Choose between:
```

Comment:
Also, I've been thinking in the context of https://bugzilla.redhat.com/show_bug.cgi?id=1478071 that this could be auto-detected based on the distro, the only issue is that customers running Ubuntu will have to set this explicitly.